pkgs/development/libraries/libbfio/default.nix

```diff
@@ -0,0 +1,29 @@
+{ lib
```

Comment:
Please move the package to `pkgs/by-name`. As it stands now, it can't be built.

Reply:
Fixed.